src/worker/worker-options.ts

```diff
@@ -1,24 +1,44 @@
-import type { KillBehavior } from '../utility-types'
-
 /**
- * Kill behavior enumeration
+ * Enumeration of kill behaviors.
  */
-export const killBehaviorTypes = Object.freeze({
+export const KillBehaviors = Object.freeze({
   SOFT: 'SOFT',
```

Comment:
We could also write JSDoc to these properties :thinking:

Reply:
mmm I don't know, I document the behavior for both on the worker options

Reply:
I want to JSDoc everything that can be a tooltip in an IDE!

![image](https://user-images.githubusercontent.com/7195563/108079595-466a8980-706f-11eb-9c78-477d82802965.png)